catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -54,10 +53,8 @@ public EnumerationValidator(final Set<String> values, final boolean ignoreCase)
 
         this.ignoreCase = ignoreCase;
 
-        this.values = Collections.unmodifiableSet(values.stream()
-                .filter(Objects::nonNull)
```

Comment:
Are you no longer worried about there being a null value in the incoming set? You'll throw an NPE on the call to `addAll` below.